corehq/apps/analytics/tasks.py

```diff
@@ -140,7 +141,7 @@ def _get_client_ip(meta):
     return ip
 
 
-def _send_form_to_hubspot(form_id, webuser, cookies, meta, extra_fields=None):
+def _send_form_to_hubspot(form_id, webuser, cookies, meta, extra_fields=None, prelogin=False):
```

Comment:
Slight preference for using webuser & extra_fields rather than adding another parameter to this function (also don't love that `webuser` is now sometimes a string and sometimes a dict).